tech/languages/fortran/fortran-installation.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Fortran
+subsection: fortran
+section: tech-languages
+version: 11.1.1
+order: 1
+description: A general-purpose, imperative computer programming language, especially suited to numeric computation and scientific computing.
+---
+# Fortran installation and usage
+
+## gfortran installation
+Fortran is a compiled language and as such it needs a compiler. To install gfortran
+compiler, simply type:
+
+```
+$ sudo dnf install gcc-gfortran
+```
+
+This command should download and install package with compiler called gfortran. To compile
+and link a program written in Fortran, simply run the `gfortran` command like this:
+
+```
+gfortran your_source.f
```

Comment:
The line should start with `$ `, as per:

https://developer-portal.github.io/contributing/write-general-writing-hints.html

Reply:
Hello, this issue has been resolved.